### DISCUSSION TITLE
Linux build in docker

### DISCUSSION THREAD

**Discussion by wipfli** [Chlumsky/msdfgen#197] [Q&A]
Thank you for this great project, it helped me learning about multichannel signed distance fields and making a first implementation draft for MSDF text rendering in the open-source map rendering engine MapLibre GL JS.

I would like to use the msdfgen executable on my linux laptop. Is the simplest way to do this spinning up a docker container and doing a clean build there?

**Comment by soimy** [comment]
I've done a Github Action CI auto-build for all platform(x64, macos-intel, macos-arm64, linux-x64, linux-arm64) at [Release page](https://github.com/soimy/msdfgen/releases)
Might be what you needed.
